### PR TITLE
Ccarfi crosschain blog

### PR DESCRIPTION
**Pull request by ccarfi**
in process, do not review yet please

### CODE REVIEW

**Comment by ccarfi** [comment]
@julien51 this is ready for review. i will NOT merge until tuesday to align with PR timeline. https://64710081fd9c21165933c161--unlock-protocol-com-staging.netlify.app/blog

File: unlock-protocol-com/blog/crosschain.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+## Life across multiple chains
+
+Technology derived from the original Ethereum experiment has gained momentum and lead to a vast array of new chains being deployed, each with their own characteristics. Under the hood, many of these chains have added the ability to execute contract calls by following the original Ethereum Virtual Machine implementation (EVM). The EVM provides a blueprint and ensures compatibility of the code that can be deployed across different chains.
+
+At Unlock Labs, we have long thought that we need to meet our community where they are, and that meant deploying the protocol on [multiple chains](https://docs.unlock-protocol.com/core-protocol/unlock/networks). 
+
+Our role is not to choose which chain/L2 on which developers choose to deploy their membership contracts — as an open protocol, that decision should be up to the community, and that decision should be permissionless.
```
Comment (julien51) [inline unlock-protocol-com/blog/crosschain.md:21]:
```suggestion
Our role is not to choose which chain/L2 on which developers choose to deploy their membership contracts — as an open protocol, that decision should be theirs, and that decision should be permissionless. 
```

File: unlock-protocol-com/blog/crosschain.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## How it works: the Unlock Owner
+
+Our design relies on a straightforward idea: the DAO on Ethereum mainnet is the authority that all contracts should listen to. When a proposal is executed, the DAO should send instructions across the bridge to all other instances of Unlock. To enable the cross-chain governance process, we have to pass contract calls and data across chains. For that, we rely on the [Connext Bridge](https://www.connext.network/).
+
+A new contract called `UnlockOwner` acts as a relay in this process, receiving the calls from the DAO across the bridge (or directly on mainnet) and passing it down to the correct contract in the protocol. It has mainly two functions: 1) upgrade the Unlock contract (core of the protocol) and 2) change settings of the Unlock contract itself (e.g. add a new template or tweak a parameter).
```
Comment (julien51) [inline unlock-protocol-com/blog/crosschain.md:37]:
```suggestion
A new contract called `UnlockOwner` acts as a relay in this process, receiving the calls from the DAO across the bridge (or directly on mainnet) and passing it down to the local Unlock contract. It has mainly two functions: 1) upgrade the Unlock contract (core of the protocol) and 2) change settings of the Unlock contract itself (e.g. add a new template or tweak a parameter).
```

**Review by julien51 — APPROVED** [review]
2 clarifications but then LG!

**Pull request merged**